src/main/java/io/vertx/jdbcclient/JDBCConnectOptions.java

```diff
@@ -0,0 +1,52 @@
+package io.vertx.jdbcclient;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.SqlConnectOptions;
+
+@DataObject(generateConverter = true)
+public class JDBCConnectOptions extends SqlConnectOptions {
```

Comment:
I think we don't really need to extend `SqlConnectOptions`, because those define many networking properties that don't make sense for this client.

Also there is no strong requirement to have a client reuse `SqlConnectOptions`

Reply:
This is required for 2 reasons:

1. The SPI to auto discover the client
2. The `QueryTracer` requires the options as that class